### ISSUE TITLE
Pointing at coincident points

### ISSUE THREAD

**Issue by mbostock** [observablehq/plot#1621]
Since the pointer transform only focuses one point at a time, if points are coincident (or nearly so), some points may not be focusable. Should the pointer transform allow focusing multiple points simultaneously, and then the tip mark could handle cycling between points, or showing multiple tips with collision detection? Or should the pointer transform support an interactive method of cycling through nearby points, say by clicking?

**Comment by mbostock** [comment]
Yep, that’s what I meant by “allow focusing multiple points simultaneously”. 👍

**Comment by net** [comment]
I think the mark should detect collisions and try to gracefully render colliding tips without further interaction, that way the solution is also applicable to non-interactive plots with static tips (such as the US states example in the docs).

**Comment by mbostock** [comment]
Yes, @net, perhaps we should have another feature request for that. Though even so that will only scale to at best ~4 tips, so we’ll still need some way to disambiguate pointing at coincident/dense points.

**Comment by net** [comment]
@mbostock if resolving the collisions through positioning fails, the mark could fall back on merging the tips into one, positioning the point of the tip at the average of the collided tips, and rendering the merged tip contents in one tip. Of course, that only scales so far too, but it would fix the US states example 🙂

```
+-------------+
| weight 54   |
| height 1.63 |
+-------------+
| weight 55   |
| height 1.63 |
+-   ---------+
  |/

**Comment by korvin89** [comment]
Hi @mbostock! First of all, thank you for your great work! :)
We are looking forward to this feature. We want to replace the current library for drawing graphs with Observable Plot, but we lack the functionality that is discussed in this issue.

What do you think about adding access to all points data with the same values of the corresponding axis via pointer? It will help to render one tip for all data(points) represented by the same value.

<details>
<summary>Here is our user case</summary>

![image](https://github.com/observablehq/plot/assets/26501073/900b15bf-805d-44d7-bc97-138aa0737090)
</details>

**Comment by Fil** [comment]
See https://github.com/observablehq/plot/pull/1671

**Comment by korvin89** [comment]
> See #1671

Great! Is it possible to stack tips into one?
[Fork of your example](https://observablehq.com/d/8f73b3deba37213b)

![image](https://github.com/observablehq/plot/assets/26501073/daf014a9-a212-4a9f-88ba-4b184866bcc8)

**Comment by Fil** [comment]
For that use case you'll probably want to group (or bin) the values by date, then compose a grouped tooltip. Here's a quick example, which is definitely not completely functional, given that there are too many lines, but hopefully enough to convey the idea:

```
    Plot.tip(
      bls,
      Plot.pointerX(
        Plot.groupX(
          {
            y: "median",
            title: data => data.map(d => d.unemployment).join("\n")
          },
          {
            x: "date",
            y: "unemployment",
            z: "date"
          }
        )
      )
    )

```
